packages/lib-classifier/src/store/WorkflowStepStore/Step/Step.js

```diff
@@ -26,7 +26,7 @@ const baseStep = types
       let isIncomplete = false
       self.tasks.forEach(task => {
         const [annotation] = annotations.filter(annotation => annotation.task === task.taskKey)
-        isIncomplete = task.required && !annotation?.isComplete
+        isIncomplete = !task.isComplete(annotation)
```

Comment:
Should this be `isIncomplete && !task.isComplete(annotation)`? If the step has a few tasks, I think this is only returning the completeness of the last task, and ignoring the others.

Reply:
Yes! Great catch, thank you